### PR TITLE
Add some headers to fill in later about Bucharest

### PR DESCRIPTION
**Pull request by maria**

### CODE REVIEW

**Comment by hfish13** [comment]
So how's Bucharest? :)

**Comment by maria** [comment]
Pretty awesome :)

**Comment by hfish13** [comment]
Foarte bine

File: maria-bucharest.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Bucharest
+
+### Places to eat
+ - Jourmey Pub
+ - Van Gogh
+ - Eden
```
Comment (mcgonagall) [inline maria-bucharest.md]: I've never been to Bucharest but I know it has a very rich and old history.  Could you be sure to add in those @maria ?

Comment (mcgonagall) [inline maria-bucharest.md]: You joined GitHub quite some time ago - what a great simple handle @maria

Comment (maria) [inline maria-bucharest.md]: We have a lot of old buildings and a Village Museum, where you can see old houses from the countryside. Also you can visit the Peasant Museum

File: maria-bucharest.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Bucharest
+
```
Comment (hfish13) [inline maria-bucharest.md]: Cel mai ieftin taxi din tara :)

File: maria-bucharest.md (hunk 1 above)
Comment (maria) [inline maria-bucharest.md]: I'm actually curious about the webinar for my colleagues who are learning GitHub :) and I make presentations for them

File: maria-bucharest.md (hunk 2 above)
Comment (maria) [inline maria-bucharest.md]: Depinde, de ora si de cat esti de atent :)

File: maria-bucharest.md [hunk 3]
```diff
@@ -1,7 +1,17 @@
 # Bucharest
 
 ### Places to eat
+ - Jourmey Pub
```
Comment (sorinr) [inline maria-bucharest.md]: ### Website:
[http://www.journey-pub.ro]
### Location:
[https://www.google.ro/maps/place/Journey+Pub/@44.4434456,26.0969724,19.5z/data=!4m13!1m7!3m6!1s0x40b1ff4e49cbd219:0x771a1073c3a77447!2sStrada+George+Enescu+25,+Bucure%C8%99ti+010303!3b1!8m2!3d44.443363!4d26.0971473!3m4!1s0x0000000000000000:0x0c82c890c156ba89!8m2!3d44.4433908!4d26.0973432?hl=en]

**Pull request merged**